### DISCUSSION TITLE
Please solve this

### DISCUSSION THREAD

**Discussion by Praveen-cyber-security** [community/community#138208] [Programming Help]
### Select Topic Area

Question

### Body

no display name and no $DISPLAY environment variable

**Comment by Y3llow45** [comment]
add:
import matplotlib
matplotlib.use('Agg')
before importing pyplot

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Y3llow45 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
